server/cli/Cargo.toml

```diff
@@ -19,11 +19,11 @@ service = { path = "../service" }
 server = { path = "../server" }
 graphql = { path = "../graphql" }
 
-anyhow.workspace = true
+anyhow = { workspace = true }
 async-graphql = { workspace = true }
 clap = { version = "3.1.8", features = ["derive"] }
 chrono = { version = "0.4.23" }
-diesel = { version = "1.4.7", default-features = false, features = ["chrono"] }
+diesel = { version = "2.1.5", default-features = false, features = ["chrono"] }
```

Comment:
I see there's a `diesel = "2.1.6"` was this a deliberate choice to go with `2.1.5`?
Would we be better to just specify `2.1` which will take the latest patch when we use cargo update?